andela/AndelaEatsUI
# #166694314  Fix editing order error on orders page

#### What does this PR do?
Fixes the error which occurs when editing some orders

#### Description of Task to be completed?
- increase end date for fetching menus
#### How should this be manually tested?
- Log into the application.
- Place an order for a day a week from now , you should be able to edit the order without an errors.

#### What are the relevant pivotal tracker stories?
- [#166694314](https://www.pivotaltracker.com/story/show/166694314)

#### Background Context
N/A
#### Screenshots (If Applicable)
N/A
